cldf/languages.csv

```diff
@@ -1,70 +1,70 @@
 ID,Name,Glottocode,Glottolog_Name,ISO639P3code,Macroarea,Latitude,Longitude,Family
-achagua,achagua,acha1250,,,,,,
-awa,awa,awac1239,,,,,,
-baniva,baniva,bani1254,,,,,,
-barasana,barasana,bara1380,,,,,,
-bara,bará,waim1255,,,,,,
-bari,barí,bari1297,,,,,,
-bora,bora,bora1263,,,,,,
-cabiyari,cabiyarí,cabi1241,,,,,,
-carapana,carapana,cara1272,,,,,,
-carijona,carijona,cari1279,,,,,,
-catio,catío,embe1260,,,,,,
-chapalaachi,cha'palaachi,chac1249,,,,,,
-chimila,chimila,chim1309,,,,,,
-cubeo,cubeo,cube1242,,,,,,
-cuiba,cuiba,cuib1242,,,,,,
-curripaco,curripaco,curr1243,,,,,,
-desano,desano,desa1247,,,,,,
-dimina,dí̵mi̵na,mala1522,,,,,,
-emberadarien,embera Darién,nort2972,,,,,,
-emberatado,embera Tadó,embe1261,,,,,,
-emberachami,embera chamí,embe1262,,,,,,
-epenasaija,epena Saija,epen1239,,,,,,
-epenabasurudo,epena basurudó,epen1239,,,,,,
-giacone,giacone,tari1256,,,,,,
-guahibo,guahibo,guah1255,,,,,,
-guambiano,guambiano,guam1248,,,,,,
-guayabero,guayabero,guay1257,,,,,,
-ika,ika,arhu1242,,,,,,
-inga,inga,inga1252,,,,,,
-jitnu,jitnu,maca1259,,,,,,
-jupda,jupda,hupd1244,,,,,,
-kakua,kakua,cacu1241,,,,,,
-kamsa,kamsá,cams1241,,,,,,
-kogui,kogui,cogu1240,,,,,,
-koreguaje,koreguaje,kore1283,,,,,,
-macuna,macuna,macu1260,,,,,,
-mirana,miraña,bora1263,,,,,,
-muinane,muinane,muin1242,,,,,,
-nukak,nukak,nuka1242,,,,,,
-ocaina,ocaina,ocai1244,,,,,,
-orejon,orejón,orej1242,,,,,,
-piapoco,piapoco,piap1246,,,,,,
-piratapuyo,piratapuyo,pira1254,,,,,,
-playero,playero,play1240,,,,,,
-puinave,puinave,puin1248,,,,,,
-paez,páez,paez1247,,,,,,
-resigaro,resígaro,resi1247,,,,,,
-secoya,secoya,seco1241,,,,,,
-siona,siona,sion1247,,,,,,
-siriano,siriano,siri1274,,,,,,
-saliba,sáliba,sali1298,,,,,,
-tanimuca,tanimuca,tani1257,,,,,,
-tariano,tariano,tari1256,,,,,,
-tatuyo,tatuyo,tatu1247,,,,,,
-totoro,totoró,toto1306,,,,,,
-tsafiquipila,tsafiqui pila,colo1256,,,,,,
-tucano,tucano,tuca1252,,,,,,
-tunebocentral,tunebo central,cent2150,,,,,,
-tuyuca,tuyuca,tuyu1244,,,,,,
-waimaja,waimaja,waim1255,,,,,,
-wanano,wanano,guan1269,,,,,,
-wayuu,wayuu,wayu1243,,,,,,
-witotominica,witoto mi̵ni̵ca,mini1256,,,,,,
-witotomurui,witoto murui,muru1274,,,,,,
-witotonipode,witoto ni̵pode,nupo1240,,,,,,
-wounaan,wounaan,woun1238,,,,,,
-yucuna,yucuna,yucu1253,,,,,,
-yukpa,yukpa,yukp1241,,,,,,
-yuruti,yurutí,yuru1263,,,,,,
+achagua,achagua,acha1250,,aca,,4.38649,-72.2005,
```

Comment:
Camel case for language idnetifiers. So this should be addressed, even if thelanguage name used in the source does not start with upper case.